packages/react/package.json

```diff
@@ -209,11 +209,11 @@
     "yaml": "2.3.4"
   },
   "peerDependencies": {
-    "@types/react": "^18.0.0",
```

Comment:
so the chevron up doesn't include 19.x? I thought it would accept everything above that 🤔

Reply:
Nope 🙃 haha, at least as far as I understand. `^` would be only for `minor` or `patch`. An alternative operator could be `>=` which I think works exactly how you described